test/module/shared_model/cryptography/crypto_model_signer_mock.hpp

```diff
@@ -74,6 +75,13 @@ namespace shared_model {
       crypto_signer_expecter->sign(signable);
     }
 
+    template <>
```

Comment:
Isn't second `template` a typo?

Reply:
Nope, because it is a template function of a template class